docker/test/integration/test_s2s.py

```diff
@@ -14,8 +14,6 @@
 # limitations under the License.
 
 from minifi import *
-from minifi.test import *
```

Comment:
Seeing import * deleted always warms my heart :)

Reply:
We still import everything, except now we don't have two separate modules that export them rather a single one. However, **now at last it is possible** to import only what is needed.

Reply:
The main reason behind my feelings on import * is that it's impossible to find the source of a given symbol when * is used across the code. So we world has just become a better place. At least our code is a bit easier to read. :)